README.md

```diff
@@ -60,6 +60,17 @@ This repository consists of these files and directories (partial):
 - `mkdocs.yml` - MkDocs recipe for the spec documentation generation. The
   inclusion of model files and the order of chapters are defined here.
 
+## Branch structre
+
+The SPDX spec repo follows the [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) workflow with the addition of support branches.
```

Comment:
Shall we link to something less critical of Gitflow?
For example https://gist.github.com/HeratPatel/271b5d2304de2e2cd1823b9b62bf43e0 (which also incudes support branches)

Reply:
Yes - I was looking for a better link - so thanks for providing a good alternative.   I was hoping to find something in Wikipedia - but the only references are to the URL I used above.

I'll update the PR.

Reply:
PR Updated